sdk/identity/azure-identity/HISTORY.md

```diff
@@ -1,5 +1,10 @@
 # Release History
 
+## 1.2.1b1 (Unreleased)
+
+- Correctly parse token expiration time on Windows App Service
+([#9393](https://github.com/Azure/azure-sdk-for-python/issues/9393))
```

Comment:
Do you want to add 2017-09-01?

Reply:
No, I think it's unimportant to users reading this.